buf.gen.yaml

```diff
@@ -10,7 +10,7 @@ managed:
     - file_option: go_package_prefix
       value: github.com/bufbuild/protovalidate-go/internal/gen
 plugins:
-  - remote: buf.build/protocolbuffers/go:v1.36.1
+  - remote: buf.build/protocolbuffers/go:v1.36.4
```

Comment:
@rodaine - Noticed this updated to 1.36.4 and we hadn't kept generated code in sync to match so pushed a separate PR to bump this and regenerate.